test/e2e/runTest.ts

```diff
@@ -0,0 +1,37 @@
+import config from "./config";
```

Comment:
this file should probably be `runClassicTest`

Reply:
It now also contains `runPlaywrightTest`.